entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+poetry run gunicorn --bind 0.0.0.0:$PORT todo_app.wsgi:app
```

Comment:
Your production image now **requires** a PORT environment variable. 

So your README instruction that says "docker run --env-file ./.env -p 5000:5000 todo-app:prod" doesn't work unless the reader has set that in .env.

You could put a default value for PORT with either:
- `--bind 0.0.0.0:${PORT:-5000}` (this is to access it in bash with a fallback value)
- Or set `ENV PORT=5000` in your Dockerfile.